app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -543,8 +542,6 @@ class HomeFragment : Fragment() {
         if (bundleArgs.getBoolean(FOCUS_ON_ADDRESS_BAR)) {
             navigateToSearch()
         } else if (bundleArgs.getLong(FOCUS_ON_COLLECTION, -1) >= 0) {
-            // No need to scroll to async'd loaded TopSites if we want to scroll to collections.
-            homeViewModel.shouldScrollToTopSites = false
             /* Triggered when the user has added a tab to a collection and has tapped
             * the View action on the [TabsTrayDialogFragment] snackbar.*/
             scrollAndAnimateCollection(bundleArgs.getLong(FOCUS_ON_COLLECTION, -1))
```

Comment:
I didn't know we had this scroll to collections functionality. It actually doesn't work :(. Unrelated to the change here though, doesn't seem to work in release for me either. Can you help me verify and then we can file an issue?

- Open tabs tray
- Select a few tabs and add to collection
- Tap on view collection

Expected: Home is opened and collection is visible
Actual: Home is opened and I am somewhere on jump back in or recent searches

If you can reproduce, let's file, as it predates the changes here.

Reply:
Confirmed that this is reproducible and not related to this change.  https://github.com/mozilla-mobile/fenix/issues/22462 created to track this.